src/IPHlpApi/IPHlpApi+MIB_TCPROW_OWNER_PID.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+#if !NETSTANDARD1_1
+    using System.Net.NetworkInformation;
+#endif
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="MIB_TCPROW_OWNER_PID"/> nested type.
+    /// </content>
+    public static partial class IPHlpApi
+    {
+        /// <summary>
+        /// The MIB_TCPROW_OWNER_PID structure contains information that describes an IP v4 TCP connection
+        /// with IP v4 addresses, ports used by the TCP connection, and the specific process ID (PID)
+        /// associated with connection.
+        /// </summary>
+        /// <seealso href="http://msdn2.microsoft.com/en-us/library/aa366913.aspx"/>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct MIB_TCPROW_OWNER_PID
+        {
+            /// <summary>
+            /// The state of the TCP connection.
+            /// </summary>
+            [MarshalAs(UnmanagedType.U4)]
+#if NETSTANDARD1_1
+            public uint State;
+#else
+            public TcpState State;
+#endif
+
+            /// <summary>
+            /// The local IP v4 address for the TCP connection on the local computer.
+            /// A value of zero indicates the listener can accept a connection on any interface.
+            /// </summary>
+            [MarshalAs(UnmanagedType.U4)]
```

Comment:
Please remove these attributes. It's unnecessary to tell .NET how to marshal integers of a fixed size, for one thing. For another, using `MarshalAs` attributes in structs block bit-blitting or using pointers to such structs, so we don't allow marshaled fields in our structs in this repo. See our [contributing doc](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#struct-field-types) for more info on this.